README.md

```diff
@@ -21,7 +22,7 @@ If you find yourself wishing for a feature that doesn't exist, you are probably
 
 # Your First Contribution
 
-Working on your first Pull Request? You can learn how from this *free* series, [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+Working on your first Pull Request? You can learn how from this _free_ series, [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
```

Comment:
I'd include something stating that each project has their own contributing guide, and if that's not sufficient they can also check this link. We want them to ask us for help but give additional resources.